generic/tests/cfg/multi_queues_test.cfg

```diff
@@ -54,6 +54,7 @@
                     queues = ${smp_fixed}
                     vhostfds_len = ${smp_fixed}
                     tapfds_len = ${smp_fixed}
+                    vt_ulimit_nofile = 8192
```

Comment:
Is this enough? I would suggest 10240.

Reply:
I tried，8192 is enough for `max_queues`. 
```bash
# cat /proc/sys/fs/file-nr
4132	0	12265804
# lsof -p `pgrep qemu-kvm` | wc -l
2145
```